# Raw Rfans Point Info.
uint16 angle      # scan angle, 0.01
uint16 rangeOne   # ehco 1, cm
uint16 rangeTwo   # echo 2, cm
char  intentOne  # 0~255
char  intentTwo  # 0~255
